### ISSUE TITLE
Deploying glpi-agent and uninstalling fusioninventory-agent using group policy and the vbs script

### ISSUE THREAD

**Issue by YohanRodriguezMDM** [glpi-project/glpi-agent#244]
### Documentation
I read it more than once

### Professional support
Yes, I know

### Is your help request related to a problem?
Yep

### Expected behavior
I expected the script to uninstall FIA and install glpi-agent.
I tested it on my computer (through group policies) and it just uninstalled FIA.

### Actions you've considered
I've tried to put `Verbose = "Yes"` on [line 135](https://github.com/glpi-project/glpi-agent/blob/develop/contrib/windows/glpi-agent-deployment.vbs#L135) of the VBS script but I don't see the output anywhere.

Nothing relevant in the event observer neither

### Additional context
W2019 server and W10 client

**Comment by g-bougard** [comment]
Hi, @YohanRodriguezMDM 
can you share your options setting ?

**Comment by YohanRodriguezMDM** [comment]
Sure, here they are : 
`SetupOptions = "/quiet RUNNOW=1 ADD_FIREWALL_EXCEPTION=1 TASKS=Deploy,Inventory,NetDiscovery,NetInventory SERVER='<removed>' TIMEOUT=600 TAG=AGGLO,DSI"
`

**Comment by g-bougard** [comment]
This seems correct. But maybe you can avoid TIMEOUT option.
Are you running the vbs from a PowerShell ? If yes, better use cmd.
Can you confirm which version you're installing ?
Finally, you can try to create an installation journal and maybe find why nothing happens by adding: `/L*V glpi-agent-msi-installation.log` to the options

**Comment by YohanRodriguezMDM** [comment]
> This seems correct. But maybe you can avoid TIMEOUT option.

Ok i removed it

> Are you running the vbs from a PowerShell ? If yes, better use cmd.

Nope, I'm not running it manually, just deploying through a "startup-script" group policy

> Can you confirm which version you're installing ?

1.4

> Finally, you can try to create an installation journal and maybe find why nothing happens by adding: `/L*V glpi-agent-msi-installation.log` to the options

Done, i'll try this.

Another question : What is the behavior of `REINSTALL=feat_AGENT` if nothing is installed ?
I'm asking because it's the default value in the VBS script [L123](https://github.com/glpi-project/glpi-agent/blob/develop/contrib/windows/glpi-agent-deployment.vbs#L123)

Thanks :)

**Comment by g-bougard** [comment]
First if nothing is installed it will just install the agent without adding `REINSTALL=feat_AGENT` to options list. So "Reconfigure" option won't even apply as there's nothing to reconfigure.
`REINSTALL=feat_AGENT` will only be added if the script detects the agent is installed and it has the same version: so this is only used to activate the reconfiguration mode as feat_AGENT is the based feature on which configuration will be managed by the MSI package.

**Comment by YohanRodriguezMDM** [comment]
> First if nothing is installed it will just install the agent without adding `REINSTALL=feat_AGENT` to options list. So "Reconfigure" option won't even apply as there's nothing to reconfigure.

Ok I misread the script.



> Finally, you can try to create an installation journal and maybe find why nothing happens by adding: `/L*V glpi-agent-msi-installation.log` to the options

I can't find log files. I'm not sure the `msiexec` was even executed.

**Comment by g-bougard** [comment]
You should probably double-check the `SetupLocation` you defined if it's not the default one.
If it's the default one, maybe there's an error while the script tries to download the MSI package from Github. Do you have an anti-virus or to proxy setup ?

**Comment by YohanRodriguezMDM** [comment]
> You should probably double-check the `SetupLocation` you defined if it's not the default one.

Files are located in my domain "sysvol" folder, I added specific permission to read and execute `.msi` files in this specific folder and I'll retry

**Comment by YohanRodriguezMDM** [comment]
Feedback from my (not so) succesful experience.
I've tried everything I could to make it work with the vbs, but it just never worked properly.
I finally modified the vbs and kept only the "uninstall fusion inventory agent" part.

Then I made 3 GPOs : 

1. Uninstall FIA through the vbs 
2. Apply Agent parameters with the admx/adml files from this repo
3. Install .msi with the built-in group policy installation

And now, it works perfectly.

Thanks for helping @g-bougard :)
